src/types/EntitiesMap.ts

```diff
@@ -0,0 +1,7 @@
+export interface Entity<T extends number | string = number> {
+    id: T;
+}
+
+export interface EntitiesMap<E = Entity> {
+    [index: number]: E;
```

Comment:
This type is wrong now. `index` can be a `string` as well (it's always casted to string anyway by the nature of what object keys are in JS, hmm). So the type is inaccurate, and the name as well (`key` suits better if it can also be a `string`).

It should be something like this:

```suggestion
    [key: T]: E<T>;
```

Or we can just remove it since `EntitiesMap` is not used anywhere in this SDK yet.

Reply:
> It should be something like this: 
>
> ```
> [key: T]: E<T>;
> ```

FYI TS will not allow this.

Reply:
Since it's not used anywhere, let's just remove it. Furthermore, most of the entities collections are arrays anyway, right? So, we don't even need this interface.

Reply:
Lukas is right, we don't need it in this repo. So feel free to remove it! :+1:

Reply:
Also there's no real need for `Entity` type *in this repo* as well.

Reply:
Removed `EntitesMap` 39a08864cbb930cdf0a4724e41b3a5f1a8f7a775